RR.App/Controllers/HRIS/AuthenticationController.cs

```diff
@@ -85,7 +84,6 @@ public async Task<IActionResult> CheckUserExistence()
                     return NotFound("User not found.");
                 }
 
-                GlobalVariables.SetUserId(employee.Id);
```

Comment:
Logic: this setUserId is done here when the user does not exist yet and the account should be finalized, i.e. the users first login. It is pulled from the database.